src/app/context/wallet/CosmosWalletProvider.tsx

```diff
@@ -1,3 +1,4 @@
+import { SigningStargateClient } from "@cosmjs/stargate";
```

Comment:
We are importing this, but it is not a direct dependency of the app (we don't have `@cosmjs/stargate` in the package.json)